### DISCUSSION TITLE
736AC0F1-5829-4CA7-B1A8-2F496CA2B236

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#110] [General]
# 736AC0F1-5829-4CA7-B1A8-2F496CA2B236

HAT-P-5, also known formally as Chasoň, is a Sun-like star about 998 light-years away in the constellation Lyra. Both this star and its gas giant planet, Kráľomoc, were given formal names by the people of Slovakia during the second NameExoWorlds campaign in 2019

https://celestia.mobi/addon?item=736AC0F1-5829-4CA7-B1A8-2F496CA2B236

**Comment by EricNelson-sketch** [comment]
It looks like a good texture for this planet after all.
